Buttons/StudentDesc.md

```diff
@@ -0,0 +1,14 @@
+# Buttons
+This week is all about adding interactivity to your web pages!
+
+## PowerPoint Presentation
+<iframe src='https://view.officeapps.live.com/op/embed.aspx?src=https://hylandtechoutreach.github.io/ucs-js/Buttons/Buttons.pptx' width='100%' height='450px' frameborder='0'></iframe>
+
+## Code-Along: Hello, Buttons!
+Follow along with the instructor to:
+- add a `<button>` to a web site
+- _define_ a function in **script.js**
+- _call_ the function by clicking the button
+
+## Self-Paced Activity: Tickle Me Elmo
```

Comment:
we shouldn't do this, but it would be fun to add an actual audio component to the site. or maybe add a challenge for that